Source/iOS/UIImage+Extensions.swift

```diff
@@ -23,4 +19,11 @@ extension UIImage {
 
     return result
   }
+
+  /// Convert to data
+  func cache_toData() -> Data? {
```

Comment:
Why do we need `cache` prefix here? It's not even public.

Reply:
It's not public, but it can cause conflicts 😬

Reply:
I don't like underscore TBH

Reply:
If it's not public I think it's fine to go without prefix.